docs/maps/mapBubble-series.md

```diff
@@ -0,0 +1,11 @@
+Map Bubble
+==========
+
+MapBuble series is a combination of the standard bubble chart series with the possibility of applying to specific parts of the map.
+
+<iframe style="width: 100%; height: 520px; border: none;" src="https://highcharts.com/samples/maps/demo/map-bubble" allow="fullscreen"></iframe>
```

Comment:
Shouldn't this be `https://highcharts.com/samples/embed/maps/demo/map-bubble`? I am not sure about results without `embed` but if you could point to another demo with a similar URL in docs then it will be ok.